cmd/mimir/config-descriptor.json

```
@@ -46,6 +46,17 @@
       "fieldType": "duration",
       "fieldCategory": "experimental"
     },
+    {
+      "kind": "field",
+      "name": "max_groups_per_user",
+      "required": false,
+      "desc": "Maximum number of groups allowed per user by which specified distributor and ingester metrics can be further separated.",
```

```suggestion
      "desc": "Maximum number of groups allowed per user. You can further separate specific distributor and ingester metrics.",
```
This is really hard to parse. Is my change still technically accurate?

---

IMO it's less accurate. I liked the previous version better.